build.gradle

```diff
@@ -5,16 +5,11 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.1.3'
+    classpath 'com.android.tools.build:gradle:3.5.3'
     classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:2.2.0'
   }
 }
 
-task wrapper(type: Wrapper) {
```

Comment:
might be worth updating this to the latest version of gradle (6.1.1)

Reply:
small thing is I had to remove `findbugs` plugin since it's been removed in v6